### ISSUE TITLE
`forum-post-countdown` doesn't work on multiple pages at once

### ISSUE THREAD

**Issue by mybearworld** [ScratchAddons/ScratchAddons#6642]
### Description

I often browse QaS by opening a bunch of unread topics in a new tab. This makes `forum-post-countdown` break though - it'll not show the countdown, and then when posting and encountering the 60 second rule, it'll reset it, so the countdown is there, but incorrect.

### Steps to reproduce

1. Open two topics (A and B) in a new tab.
2. Post on A.
3. Go to B, you'll see the countdown not being there.
4. Try to post on B.
5. You'll encounter the 60 second rule, and the countdown will begin counting from 60.

### Demonstration(s) and additional context

I know this is expected behavior as it was coded like that, not really sure where to put it...

### Environment

Browser: Edge 116
Scratch Addons version: 1.34.1
Operating system: Windows

**Comment by DNin01** [comment]
Yeah, we need to check if there's a countdown in progress when the user switches tabs.

**Comment by WorldLanguages** [comment]
I think the "Follow topics after posting" addon knows whether the post was sent successfully or not. We could check that if we haven't already.

To sync tabs, the best way is probably the window.onstorage event: https://developer.mozilla.org/en-US/docs/Web/API/Window/storage_event

**Comment by Samq64** [comment]
Did #8199 fix this?
